### PR TITLE
Loyalty Implants removed from cargo

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
By which I mean security starts with their 8 implants, and that is it. No more extra from cargo.

I think that the order implants/tase everyone/implant everyone metagame for conversion modes is very boring. Cutting security off cold turkey would also be bad though (or maybe not so much bad as anger inducing), so they still have a few spare to start the round.

I plan to test merge this. Admins are obviously going to have to play along and give security some extra leeway during conversion modes, and crew some extra leeway to kill security if they decide to just murder everyone.

EDIT: Maybe I should do this the other way around and have them be a cargo crate but not something they start with? Or just drop the whole thing, it'll probably just cause tears.

### CODE REVIEW

**Comment by Aloraydrel** [comment]
wew

**Comment by Bluespace** [comment]
"he may or may not be a rev but we have no implants so we should just beat his head in anyway"

**Comment by Dawgas** [comment]
doesn't fix the problem about team conversion modes for all intents and purposes playing the same and therefore doesn't fix them being boring as shit to play for or against

remove conversion based antag modes 2016
drink bitch tears 2016 (b-but all my hard work going down the drain because it was shit!)

**Comment by ShadowDeath6** [comment]
This is gimping security even more, who usually gets wiped out in its entirety anyway. 

>_Remove conversion based antag modes 2016_

**Comment by KorPhaeron** [comment]
People rioted when conversion cult was removed. People love conversion modes.

**Comment by KazeEspada** [comment]
That's because everyone gets to be an antag... unless you are a head or security.

**Comment by CPTANT** [comment]
This leaves clubbing everyone to death the only viable option during rev/gang.

**Comment by Durkel** [comment]
Looking forward to being bolted into the brig every rev/cult/gang mode.

**Comment by GunHog** [comment]
>Looking forward to being spaced every rev/cult/gang mode.

I fixed that for you.

**Comment by CPTANT** [comment]
Actually security just murdering every non implanted person on the station might actually be kinda interesting.

**Comment by KorPhaeron** [comment]
Security vs gangs vs neutral crew vs angry AI (why didn't you change its laws?) in an all out fight to the death will probably be more interesting than "secure cargo lets implant everyone on station"

**Comment by Xhuis** [comment]
:-1: :-1: :-1: :-1: :-1:

**Comment by lzimann** [comment]
This is worst, having more implants from cargo at least prevents security from killing very single person after they spent their initial implants.

**Comment by tkdrg** [comment]
"Just remove conversion"

but once I was done

"Wow stop removing conversion why do you hate fun"

On topic, this is good because it reduces sameyness for conversion modes and makes picking who to implant a much more interesting decision

**Comment by AdamElTablawy** [comment]
But it sucks for everyone else in the round because sec is going to get lynched by the entire crew even if it's not rev and if someone makes a false declaration of rev sec kills everyone.

and then there are nine million bored ghosts in deadchat every round and the change will be unanimously hated for getting them killed.

**Comment by CosmicScientist** [comment]
:+1: for test merge. No reason to not question parts of the game.

**Comment by Zilenan91** [comment]
We probably have too many conversion spam modes for our own good. This somewhat helps the tedium but I still feel like it doesn't solve the problem.

**Comment by Steelpoint** [comment]
I too enjoy seeing Security execute anyone and everyone they capture since they have no method of reliably de-converting someone.for good.

It doesn't take a genius to see the outcome of this change.

**Comment by Zilenan91** [comment]
What if we had a test where we disabled all team antags but rev for a while to see what this change does.

**Comment by Steelpoint** [comment]
People bitch enough about Security during conversion based rounds when a non-antagonist gets caught up by Security and is thrown into Cargo for 5 to 10 minutes, what do you think people are going to think when instead of confinement its now on the spot execution.

**Comment by Zilenan91** [comment]
That's not really a failing of security though. That's a failure in how the mode has been designed. Cult and Gang have some huge problems with this and quite badly (gang moreso) need updates.

**Comment by optimumtact** [comment]
This validates my desire to murder security the moment any conversion deconversion antag round is known

**Comment by PKPenguin321** [comment]
ehhhh
:-1: 
i know that sec capturing everybody with no implanting and holing out in cargo is kinda shit, but at least it gives antags a chance to fight back and escape. with this, sec will just gun everybody down, no cuffs required. less engaging

**Comment by YakumoChen** [comment]
also heavily nerfs security because having an army of valid-happy implanted crew to help you out was a big factor in security winning conversion rounds.

**Comment by Stickymayhem** [comment]
I actually find security holing up and setting up a defensive position somewhere kind of fun. Either they get the secure brig and have to send troops to make loyalty implant supply runs which the revs can cut off, or use a less secure area in cargo and combat the tide.

That said I'm always a supporter of my security bashing peoples fucking heads in so it might be good to test this for a couple days.

**Comment by SweaterKittens** [comment]
:-1:

I'm going to agree with what a lot of people are saying here in that it will just encourage rampant violence (as if we don't already have enough of that in these roundtypes).

While running to secure cargo may be "boring", it's literally the only /chance/ that security has outside of genociding the entire goddamn station. And in practice, I see securing cargo and trying to implant everyone work about... 25% of the time. Maybe less. Security and loyal crew are not designed to win these rounds, and by removing their only method of fighting back, it's just going to get a ton of people (and because they can't prove that they're gangers/revs or not) /innocent/ people killed. 

I think if securing cargo was a foolproof and auto-win scenario for security, there might be a need for a change, but right now they lose so fucking much. I /hate/ gang with a fucking passion, report it to security first chance I get and do my damndest to try and get the captain and the HoS to secure cargo and start implanting. And as I said, even with me pushing as hard as I can, it works less than 1/4 of the time. 

TL;DR Gangs and revs win all the time anyway, don't take sec's only reasonable and non-shitty way of fighting back off the table.

**Comment by KorPhaeron** [comment]
@SweaterKittens 

http://www.ss13.eu/tgdb/tg/latest_stats.html#mode_revolution

We unforunatley do not have reliable stat tracking for gang victories, but the rev/security split is actually 50/50, not 75/25

>I hate gang

I guess that is where any hope of common ground falls apart then, because a lot of my motivation is because I enjoy gangs fighting and don't like seeing security stomp it out as soon as they see a gang tag

**Comment by SweaterKittens** [comment]
@KorPhaeron 

Honestly I'm not surprised by those stats, because I feel like they're heavily weighted towards a gang/rev loss by two things:
1. Gang/Rev head gets caught out within 5 minutes of roundstart, before anything can happen and it either turns into extended or the gangs/revs have a slim-to-none chance of making anything happen
2. The round drags on so long, that the gangs all kill each other off to a degree that no one can drop a dominator without it getting merc'd, and the shuttle finally leaves since the station is at a stalemate.

Obviously the second one doesn't apply to rev, but I've seen plenty of ERTs, admin events, or even just a few robust players kicking the shit out of the remaining Rev heads with zero interaction from security/implanting. But basically what I'm saying is that I don't feel as though the win/loss statistics as a whole accurately reflect how effective a single element is at combatting gang/rev. In my experience, securing cargo and force implanting has to be done incredibly quickly, with good coordination and robust and effective arresting, or cargo will get bombed/overrun, sec will get converted/killed, or there will just be too many gang members to handle. And oftentimes, it fails. 


Unfortunately you're correct, in that while you enjoy watching gangs duke it out, I enjoy not being forced to involve myself in what I consider to be a very unfun team deathmatch gamemode. I also have the majority of my traitor prefs off, because I like playing a 'loyal' crewmember. However, while we lack common ground in that aspect, I think we're both on the same page in that we don't want to see a one-sided stomp. I don't like seeing security and loyal crew shithoused before they've had a chance to react, and you don't like seeing the gangs stomped out before they can use their pen more than once. Personally, I feel like right now the game has those two in a good balance, and unless someone does something extraordinarily stupid right off the bat, the gangs have a very good chance to win, even if security organizes a decent response.

**Comment by Steelpoint** [comment]
While stats only give you so much information I can say, based off of the stats page, that Rev is one of the most balanced game modes we have IN TERMS of win rate.

Win rates remain mostly around the 50/50 mark irrespective of population nor the round length.

Of course there could be one to one hundred different reasons why one side loses, maybe Sec ONLY wins when the rev heads are incompetent or vice versa.

**Comment by Dawgas** [comment]
stat tracking isn't even legitimately reliable in seeing what's good or bad, with how the rest of the factors of the game interact, including player skill and other code changes.

That has probably been said already, but sometimes it feels like that needs to be beaten into people.

**Comment by Robustin** [comment]
I vote drop the whole thing.

Removing the roundstart implants makes it a nightmare to test for Revs/Gangs, which are very momentum based rounds. Removing cargo implants means that Sec losing gangs is virtually guaranteed unless Security goes full Gestapo and starts exterminating entire departments. 

What is it with coders not wanting subtlety in their solutions? Implant spam could easily be fixed by removing the HOS's backup implants and increasing the cost at cargo. At some point increasing implant cost (they now cost 5,000 points!) is equivalent to removing them from cargo altogether, so why not just increase cost to a lesser degree and see how the reduced supply affects conversion rounds.

**Comment by Steelpoint** [comment]
In what hell is removing any way to get more Loyalty Implants from cargo a acceptable change?

EIGHT implants for a station that can range from thirty to eighty players is going to do jack shit.

Do I honestly need to explain how bad of a idea this is? From Gangsters needing TWO implants to deconvert, making you guaranteed to lose a implant when you try and implant a Head Rev, and don't get me started on people who get implanted but then run off to get their implant removed.

If your goal is to encourage Security to go full SS-Station on the crew then this PR does it. Because the outcome here is that Security WILL execute absolutely everyone on the station. It'll be roaming literal death squads of officers killing anything and everything.

People already hate it when they get executed by Sec (or the revs/whatever), imagine the hatred when Security executes 90% of the station?

**Comment by KorPhaeron** [comment]
>If your goal is to encourage Security to go full SS-Station on the crew then this PR does it. Because the outcome here is that Security WILL execute absolutely everyone on the station. It'll be roaming literal death squads of officers killing anything and everything.

More exciting than a line of people bucklecuffed to chairs in cargo every rev round

**Comment by KorPhaeron** [comment]
>EIGHT implants for a station that can range from thirty to eighty players is going to do jack shit.

Good point the implants should scale with round pop

**Comment by KorPhaeron** [comment]
>from Gangsters needing TWO implants to deconvert

I wouldn't mind making them only need 1 if security had limited implants

**Comment by Steelpoint** [comment]
There are better ways to balance Loyalty Implants then going from 0 to 100 and removing them from cargo.

Hell, Oldman gave a good suggestion. Just remove the excess HoS implants and buff the cargo cost slightly.

**Comment by Steelpoint** [comment]
Do implants even need balancing?

If Sec has got everyone lined up in Cargo for deconversion then what would happen if there were no more implants? It would be a line of dead people and Sec will still be winning if its at that point.

All your doing is removing a somewhat edge scenario to punish Sec's ability to counter any conversion Rev in the remaining % of scenarios.

**Comment by KorPhaeron** [comment]
Right now if security sees one gang tag on a wall they will seize cargo and order a massive amount of implants, then implant everyone. This sucks for the gangs who get stomped before they've even done anything, this sucks for cargo who doesn't get to do their job, and it sucks for everyone else (sec included) who are now playing extended.

I don't think security will suddenly gun down the entire station upon seeing spraypaint on the wall.

Sure they'll start killing people once it reaches a crisis point, but at least the crisis point will get reached instead of the round failing to ever really start.

**Comment by optimumtact** [comment]
I agree with Kor

- [x] Kor
- [ ] Not Kor

**Comment by Supermichael777** [comment]
You underestimate the shittyness of the average HOS

**Comment by optimumtact** [comment]
the ban button is a mouse click away

**Comment by Supermichael777** [comment]
No this validates the discison because "we dont get a choice" it forces the issue

**Comment by KorPhaeron** [comment]
>we don't have a choice but to kill literally everyone once we see spraypaint

??

**Comment by Dorsisdwarf** [comment]
A reminder that sec can deconvert revs by bashing them in the face.

**Comment by Robustin** [comment]
Scaling roundstart implants and increasing cargo cost seems like the intelligent solution here.

Increase the cost enough so you can't bring in 3+ crates if cargo's done its basic duties. Testing it at 80 or even 100 seems logical, probably too harsh for highpop conversion rounds, but we'll see.

**Comment by Steelpoint** [comment]
So long as scaling is done right even on high pop it can be feasible.

But just outright removing any ability to restock Loyalty Implants is a bad design decision.

**Comment by Zilenan91** [comment]
It's really not. It makes it so there's actual conflict in the round rather than sec just implanting everybody and calling the shuttle.

**Comment by Steelpoint** [comment]
Oh you got the conflict part of your argument down pat.

Meaning instead of converting everyone, sec will just use their four to X implants on the first few people before they switch to just executing everyone, and anyone, they find.

Do you really need to coddle the round antagonists in this manner? If the antagonists cannot muster the manpower to counter Security's attempts to implant the crew then they don't deserve to win.

That's like complaining about how Security has no chance to do anything in a round and how its boring that the rev's/gang/etc just implant everybody and end the round.

**Comment by Incomptinence** [comment]
So instead security will kill the cargo techs then order more guns? 

At least give sec more weapons so they are better suited to killing the entire station.

**Comment by Dorsisdwarf** [comment]
Reminder that sec can beat revs in the head to deconvert

**Comment by Incomptinence** [comment]
@Dorsisdwarf 
Which is random with no indicator other than reconversion.

Every innocent wrongly fingered will be identical to a fucking rev head. People who randomly don't de-convert will be identical to the previous two groups. So now sec invades medical and kills everyone there? Wow beating deconversion so goooood.

Also gangers will just be killed. I have had it with your mindless copy paste deflections.

**Comment by Steelpoint** [comment]
Incompitence has it, all this change will do is massively incentivise security to execute anyone who does not have a loyalty implant when they run out of their initial supply. 

While I'm certain I'm repeating myself, but people right now complains a LOT when security executes people in conversion game modes. I loath to see what the reactions will be when security systematically begins to slaughter the station.

**Comment by GunHog** [comment]
The slow influx of makeshift "ghetto" devices and weapons means that the average crew member will have the means to make a stand and fight back.

**Comment by duncathan** [comment]
@KorPhaeron so do you want this merged or what

**Comment by Steelpoint** [comment]
Will this change affect the pricing of everything else? Since the view of this change lists the pre-goon cargo costs.

Also this is still a horrid suggestion.

**Comment by Metacide** [comment]
What an awful idea.

At least listen to Robustin on balance issues. Why do we seem to be implementing every idea from the shitpost ideas thread? Kor and HG will just keep memeing everyone unless we stop them now.

**Comment by Dorsisdwarf** [comment]
@Incomptinence 

Are... 

are you okay?

**Comment by Antimattercarp** [comment]
So the problem is that security is converting people too much and the conversions are too powerful, right?

So why are we talking about neutering secs ability to convert instead of buffing team based antags ability to deconvert?

I mean shoot, doesn't gang already have that functionality in one of their goodies.

**Comment by Incomptinence** [comment]
@Antimattercarp 
I have seen implant removal and clone conversion used a few times by revs and gangers.

So yeah since security conversion is totally item based medical can undo it pretty easily.

**Comment by ExcessiveUseOfCobblestone** [comment]
Saying someone is going to have a stunbaton applied to their head repeatedly without knowing if they are actually rev/not implies you are just doing the same tactic of abducting people as sec to deconvert without actually seeing if they've done wrong [which is exactly what this PR is trying to pull away from], or that the person has not done something greytide-like during a rev/gang round where they deserve beatings in the head anyways.

**Comment by ChangelingRain** [comment]
wew
